### ISSUE TITLE
JSON letter experience in the mobile app

### ISSUE THREAD

**Issue by TKDickson** [department-of-veterans-affairs/va-mobile-app#6215]
# Epic Status Overview
### Problem Statement:
_High level description of the epic and the problem we are trying to solve_
Enable Veterans who are using assistive technology (AT), like a screenreader, to read letter content in the app (not just rely on PDFs)

Currently letters show in the app in a PDF format and are not readable by screen readers. The PDF is not accessible for screen readers. The goal of this work is to show letters as an HTML rendered version of a JSON file. The Veteran should still have the option to view the PDF version of their letter.

### Why is this Important?:

- This will enable Veterans who use assistive technology to more easily understand their benefits & explain them to others like employers or housing managers (like everyone else who uses letters).

### Definition of Done:
What set of criteria must be met for this epic to be considered complete?

- Veteran is able to read letters in app using assistive technology

### Success Criteria:
What measurable outcomes will be used to indicate that this epic has successfully delivered value (i.e. - what is the impact)?

- 

### Product Brief:
- [link to GitHub Product Brief]

### Monthly Update for Stakeholders

 
|Date|  Progress & Key Accomplishments| Issues/Risks/Blockers|
|-------| --------|---------|
|MM/DD/YY| 
|MM/DD/YY|
|MM/DD/YY|
|MM/DD/YY|

### VA Stakeholders

- 
 
### Reference Links
List links related to epics, pitch deck, one pager, product folder, etc.

**Comment by timwright12** [comment]
@rtwell @BeccaTupaj Thoughts on reactivating this work?

**Comment by rtwell** [comment]
I'm all for it—let me get it in the queue with the other items I have in the hopper

**Comment by timwright12** [comment]
awesome, I think this would be great

**Comment by BeccaTupaj** [comment]
Sounds good! I'll use this ticket as the epic, and we can add user stories when we are ready to start. @rtwell I'll add this epic ticket to Global Mobile.

**Comment by BeccaTupaj** [comment]
Per @timwright12 this will involve both front and backend work, but some backend work has already been completed.

**Comment by NatalieDavied** [comment]
@BeccaTupaj I have created a new [epic](https://github.com/department-of-veterans-affairs/va-mobile-feature-support/issues/854) on the MFS board and added it to the Global Mobile Board. This epic is good to close.
